Source/m_menu.c

```diff
@@ -3624,6 +3625,9 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Default compatibility", S_CHOICE|S_LEVWARN, m_null, G_X,
    G_Y4 + general_comp*8, {"default_complevel"}, 0, NULL, default_compatibility_strings},
 
+  {"Show ENDOOM", S_YESNO, m_null, G_X,
```

Comment:
I only want to see ENDOOM for PWADs that provide their own lump, so please turn this into a choice of three: "on", "off", PWAD only".